zk-token-sdk/src/encryption/elgamal.rs

```diff
@@ -622,6 +656,7 @@ mod tests {
         super::*,
         crate::encryption::pedersen::Pedersen,
         solana_sdk::{signature::Keypair, signer::null_signer::NullSigner},
+        std::fs::{self, File},
```

Comment:
Is this needed?

Reply:
These seem to be needed for the tests https://github.com/solana-labs/solana/pull/31496/files#diff-7d059813b85f149efcc5236cf7f8c29645b18bb9787cdaa05cec84711b06a8a6L811-L821?

Reply:
Oh, I see. Sorry!